Introduce a change to start review online (#35)
JulianKarlBauer/mechkit: paper/paper.md

The Python package `mechkit` is a toolkit for researchers
in the field of continuum mechanics and material modeling.
`mechkit` contains methods and operators
for common tasks concerning tensor algebra and tensor notation.

In the context of engineering applications in three spatial dimensions, deformations and stresses in solids are
usually described by second-order tensors.

> Introduced as linear mappings between the second order tensors of observed deformations and causal stresses,

Reviewer: ```suggestion
As linear mappings between observed deformations and possibly causal stresses,
```

The methods and operators for the description of linear elasticity and material modeling are mathematically simple and can be expressed in a compact manner.
Due to this relative simplicity, they are usually implemented independently by each scientist for use in their studies.

> Consequentially, unlike in other research fields such as physics and computer science, there is no library for the common methods and operators of linear elasticity and modeling to this day.

Author: fixed typo in line 61

Reviewer: thx

> This presents a major obstacle with regard to the exchange and reliability of research code as well as the interpretation and comparison of results.

Author: The two sentences read repetitious.

> The main goal of `mechkit` is reusable research code that increases the reliability of results and

Author: "the project" has not been used in conjunction with `mechkit` before in this publication

Consequentially, for even a small number of different notations, the implementation of all corresponding translation methods is not feasible.
Therefore, `mechkit` does not necessarily directly translate one notation into another.
Instead, in the case of the translation of second- and fourth-order tensors,

> `mechkit` determines the shortest path between source and target in the set of implemented notations as illustrated in

Author: This is a really neat feature and we should consider to emphasize this even further.
